invenio/base/config.py

```diff
@@ -75,24 +75,29 @@
     'invenio.legacy.websubmit',
 ]
 
-CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
+_CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
```

Comment:
`invenio/ext/logging/__init__.py:deprecated` we could move it to `invenio/utils/deprecation.py`. WDYT?